lib/infrastructure/converters.dart

```diff
@@ -16,3 +17,33 @@ String? nullableDateTimeToString(DateTime? dateTime) {
   }
   return null;
 }
+
+List<ChildAge> childrenAgesFromString(String childrenAgesStr) {
+  List<ChildAge> childrenAges = [];
+  childrenAgesStr.replaceAll("[", '');
+  childrenAgesStr.replaceAll("]", '');
+  var splitAges = childrenAgesStr.split(',');
+  for (String ageStr in splitAges) {
+    ChildAge childAge = ChildAge();
+    childAge.age = int.parse(ageStr);
+    childrenAges.add(childAge);
+  }
+  return childrenAges;
+}
+
+String childrenAgesToString(List<ChildAge> childrenAges) {
+  String childrenAgesStr = '[';
+  for (ChildAge age in childrenAges) {
+    childrenAgesStr += age.toString() + ",";
+  }
+  childrenAgesStr += "]";
+  return childrenAgesStr;
+}
+
+
+String? nullableChildenAgesToString(List<ChildAge>? childrenAges) {
+  if (childrenAges != null) {
+    return childrenAgesToString(childrenAges);
+  }
+  return null;
+}
```

Comment:
To zwykła tablica obiektów składających się z pojedynczego inta.
Po co dziergać taki customowy parser?
Nie ogarnie tego json serializable out of the box? 😅

Reply:
Sama adnotacja JsonSerializable wywalała błąd
`The argument type 'List<dynamic>?' can't be assigned to the parameter type 'List<ChildAge>?' • lib/models/guest.g.dart:16:17 • argument_type_not_assignable
  error • The method 'fromJson' isn't defined for the type 'ChildAge' • lib/models/guest.g.dart:17:33 • undefined_method`

więc chciałam zrobić szybkiego fixa, ale prawda że to brzydko wygląda :p Zobaczę czy zadziała dodanie `factory ChildAge.fromJson`, czy trzeba to będzie jakoś inaczej zrobić